doc/release/release_dev.rst

```diff
@@ -22,7 +22,7 @@ New Features
 Improvements
 ------------
 
-
+- qtpy (MIT) now an optional dependency for skivi to run properly.
```

Comment:
I cannot push to your branch, but I'd like to reword this to something like:
```
- ``skivi`` is now using ``qtpy`` for Qt4/5 compatibility (a new optional dependency). 
```
before merging.